frontend_lib/src/action.async.js

```diff
@@ -24,6 +24,14 @@ export const postNewComment = (apiUrl, workspaceId, contentId, newComment) =>
     raw_content: newComment
   })
 
+export const deleteComment = (apiUrl, workspaceId, contentId, commentId) =>
+  baseFetch('DELETE', `${apiUrl}/workspaces/${workspaceId}/contents/${contentId}/comments/${commentId}`)
+
+export const putNewComment = (apiUrl, workspaceId, contentId, commentId, newComment) =>
```

Comment:
`New` makes me think it will create a comment, `updateComment`?

Reply:
or `putComment`

Reply:
will fix